etl-pipeline/test_newrelic.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+"""
+Test script for New Relic custom events
+"""
+import os
+import sys
+import time
+import newrelic.agent
+
+def main():
```

Comment:
I think we should throw this under `scripts`!

Reply:
Oh whoops I forgot to remove this. Do you think we want to include it?